test/setup/setupLanguage.ts

```diff
@@ -30,9 +30,10 @@ const lv = {
 // de_DE.json
 // lv.json - mock version with few translations, used to test fallback translation
 
-type Translations = Record<string, Record<string, string> | string>;
+type Translation = Partial<Record<"one" | "other", string>> | string;
```

Comment:
I see "zero" in some places

Reply:
Turns out its easier to handle these strings before they become nested from a find-and-replace perspective, so https://github.com/matrix-org/matrix-react-sdk/pull/11417 will handle that and then will update this PR